### ISSUE TITLE
Module not found in versions 1.2.x

### ISSUE THREAD

**Issue by janikarh** [timothycrosley/portray#23]
This just in: Worked fine in v1.1.0, v1.2.0 and v1.2.1 gives `Module not found: [module name given in toml]`.

**Comment by timothycrosley** [comment]
@janikarh can you confirm that the module(s) listed are available from within the same venv as portray?

**Comment by janikarh** [comment]
@timothycrosley In this case the module is just a single app.py file on the project root. Only thing which changed between, is that I upgraded `portray`version.

**Comment by timothycrosley** [comment]
Thanks for the additional information! I have some ideas I'll look into. 

I know this is almost certainly related to the switch from pdoc3 to pdocs: https://timothycrosley.com/project-3-pdocs however as detailed in the blog, it's a necessary change for licensing reasons

**Comment by timothycrosley** [comment]
Confirmed this is a difference between how pdocs and pdoc3 handle a module available in the ran directory but not on the path:

```bash 
timothyedmundcrosley@pop-os:~/Projects/portray/test_non_path_project$ pdocs server my_program
Module not found: my_program
timothyedmundcrosley@pop-os:~/Projects/portray/test_non_path_project$ pdoc3 my_program
Module my_program
=================

Functions
---------

    
`my_method()`
:
```

Fix incoming

**Comment by timothycrosley** [comment]
This should be fixed in the just released 1.2.4: https://timothycrosley.github.io/portray/CHANGELOG/#124-september-5-2019.

Thank for your patience, and please let me know if you still experience the issue in the new release.

Thanks!

~Timothy

**Comment by janikarh** [comment]
I can confirm that the fix worked in my test cases.
